packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/s3/S3Service.java

```diff
@@ -526,21 +533,24 @@ public ResponseEntity<StreamingResponseBody> getDownloadStream(final String buck
 			}
 		};
 
+		final CacheControl cacheControl = CacheControl.maxAge(24, TimeUnit.HOURS).cachePublic();
```

Comment:
Should we put that `24` in the application.properties @dvince2?

Reply:
I'm always a fan of getting rid of magic numbers - either as a static final at the top or in application.properties